src/main/java/to/us/resume_builder/ApplicationConfiguration.java

```diff
@@ -0,0 +1,83 @@
+package to.us.resume_builder;
+
+import com.google.gson.Gson;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class enables the use of a configuration file with the program.
+ */
+public class ApplicationConfiguration {
+    private static ApplicationConfiguration instance = null;
+    private Map<String, Object> configuration;
+
+    /**
+     * Construct the <code>ApplicationConfiguration</code> singleton instance.
+     */
+    public ApplicationConfiguration() {
+        // Read the configuration file
+        Gson gson = new Gson();
+        try {
+            configuration = gson.fromJson(new BufferedReader(new FileReader("config.json")), Map.class);
```

Comment:
Based off of what Cerny mentioned on Thursday, we may want to wrap the BufferedReader to prevent names with special characters from getting wrecked.

Reply:
UTF-8 encoding now specified. On modern systems, it would use it by default, but I specified UTF-8 just to be safe.